local-tests/chainrunner/node.py

```diff
@@ -46,7 +52,23 @@ def stop(self):
     def purge(self):
         """Purge chain (delete the database of the node)."""
         cmd = [self.binary, 'purge-chain', '-y'] + self._stdargs()
-        subprocess.run(cmd, stdout=subprocess.DEVNULL, check=True)
+        subprocess.run(cmd, stdout=subprocess.DEVNULL)
+
+    def rpc_port(self):
+        """Return RPC port for this node. The value is taken from `flags` dictionary.
+        Raises KeyError if not present."""
+        port = self.flags.get('rpc_port', self.flags.get('rpc-port'))
+        if port is None:
+            raise KeyError("RPC port unknown, please set rpc_port flag")
+        return port
+
+    def ws_port(self):
+        """Return WS port for this node. The value is taken from `flags` dictionary.
+        Raises KeyError if not present."""
+        port = self.flags.get('ws_port', self.flags.get('ws-port'))
```

Comment:
meh, do we need such fallback? we have to pass `ws_port` to `set_flags()` anyway so `ws-port` is never gonna happen

Reply:
oh, okay, in Python you can pass kwargs like `**{"ws-port": 9933}`, but I think that such people are asking for troubles

Reply:
I'm just trying to keep the `-`/`_` compatibility everywhere. No one will probably ever need that, but it's better to stay consistent